x-pack/legacy/plugins/apm/common/projections/services.ts

```diff
@@ -9,13 +9,13 @@ import { SERVICE_NAME, PROCESSOR_EVENT } from '../elasticsearch_fieldnames';
 import { rangeFilter } from '../../server/lib/helpers/range_filter';
 
 export function getServicesProjection({ setup }: { setup: Setup }) {
-  const { start, end, uiFiltersES, config } = setup;
+  const { start, end, uiFiltersES, indices } = setup;
 
   return {
     index: [
-      config.get<string>('apm_oss.metricsIndices'),
-      config.get<string>('apm_oss.errorIndices'),
-      config.get<string>('apm_oss.transactionIndices')
+      indices['apm_oss.metricsIndices'],
+      indices['apm_oss.errorIndices'],
+      indices['apm_oss.transactionIndices']
```

Comment:
Each config options can contain a comma separated list of several options. This is mostly used for CCS like `apm-*, *:apm-*`:

```
index: ['apm-*, *:apm-*', 'apm-*, *:apm-*', 'apm-*, *:apm-*']
```

Can you make sure that queries work with comma-separated lists?